{"status": "broken", "uuid": "4fd42e67-a0b6-478c-9768-376cb4e6626c", "labels": [{"name": "feature", "value": "地址管理"}, {"name": "tag", "value": "@pytest.mark.usefixtures('userId')"}, {"name": "suite", "value": "test_set"}, {"name": "subSuite", "value": "TestAddress"}, {"name": "parentSuite", "value": "testcase"}, {"name": "host", "value": "DESKTOP-M0PGOTF"}, {"name": "thread", "value": "13804-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython2"}, {"name": "package", "value": "testcase.test_set"}], "stop": 1629252153649, "testCaseId": "4089744c8e6baa011914ce6a3fb6df24", "historyId": "9fcb40e738241c4e06bdab3eeb11bcc7", "start": 1629252153649, "fullName": "testcase.test_set.TestAddress#test_addr_update", "statusDetails": {"message": "TypeError: 'NoneType' object has no attribute '__getitem__'", "trace": "@pytest.fixture(scope='session')\n    def userId():\n>       userId = Login().yt_login('易通')['data']['userId']\nE       TypeError: 'NoneType' object has no attribute '__getitem__'\n\ntestcase\\conftest.py:23: TypeError"}, "name": "修改地址接口"}